src/App.js

```diff
@@ -1,12 +1,56 @@
-import React from 'react';
+import React, {Component} from 'react';
 import './App.css';
+import Modal from './components/PokemonChoice';
```

Comment:
You should `import Modal from './components/Modal` or `import PokemonChoice from './components/PokemonChoice'` but not a mix of the 2.